relayer/ocr2/contract_transmitter.go

```diff
@@ -43,9 +49,23 @@ func NewOCRContractTransmitter(
 		senderAddress:      senderAddress,
 		transmissionsCache: transmissionsCache,
 		lggr:               logger.Named(lggr, "OCRContractTransmitter"),
+		transmitterOptions: &transmitterOps{
+			excludeSigs:      false,
+			ethereumKeystore: false,
+		},
 	}
 }
 
+func (oc *contractTransmitter) WithExcludeSignatures() *contractTransmitter {
+	oc.transmitterOptions.excludeSigs = true
+	return oc
+}
+
+func (oc *contractTransmitter) WithEthereumKeystore() *contractTransmitter {
+	oc.transmitterOptions.ethereumKeystore = true
+	return oc
+}
```

Comment:
This public methods (+ the constructor) should return a public interface or a struct, not an internal struct:

```suggestion
func (oc *contractTransmitter) WithExcludeSignatures() ContractTransmitter {
	oc.transmitterOptions.excludeSigs = true
	return oc
}

func (oc *contractTransmitter) WithEthereumKeystore() ContractTransmitter {
	oc.transmitterOptions.ethereumKeystore = true
	return oc
}
```